website/docs/r/secret.html.markdown

```diff
@@ -8,7 +8,7 @@ description: |-
 
 # Resource: secrethub_secret
 
-This resource allows to write secrets at a given path, if the path is exists already then the resource will write a new version of it.
+This resource allows writing secrets at a given path, if the path already exists then the resource will write a new version of it.
```

Comment:
I think `to write` is correct, isn't it?

Reply:
I think both are correct, but personally:
`This resource allows writing secrets...` or
`This resource allows you to write secrets...`
 sound better than
`This resource allows to write secrets...`

Reply:
I like this one a lot: `This resource allows you to write secrets`.
By using you, it focuses on the user.

Reply:
Good point. I will use that phrasing.